src/plugins/util/i18n.ts

```diff
@@ -1,9 +1,46 @@
 import Map from '@dojo/shim/Map';
-import { CallExpression, Identifier, Program, VariableDeclaration, VariableDeclarator } from 'estree';
+import { CallExpression, Expression, Identifier, Program, SequenceExpression, SpreadElement, VariableDeclaration, VariableDeclarator } from 'estree';
 import * as path from 'path';
 import { getBasePath, isRelative, mergeUnique } from './main';
 import { extractArrayValues, getNextItem, isShadowing } from './parser';
 
+/**
+ * @private
+ * Determines whether the provided expression is a CallExpression.
+ */
+function isCallExpression(expression: any): expression is CallExpression {
+	return expression.type === 'CallExpression';
+}
+
+/**
+ * @private
+ * Determines whether the provided expression is a SequenceExpression.
+ */
+function isSequenceExpression(expression: any): expression is SequenceExpression {
+	return expression.type === 'SequenceExpression';
+}
+
+/**
+ * @private
+ * Resolves the callee and arguments for the provided expression.
+ *
+ * @param expression An AST expression node.
+ * @return An object containing the callee and arguments if the expression is a call expression.
+ */
+function getCallExpressionData(expression?: any): { callee?: Identifier, args?: Array<Expression | SpreadElement> } {
+	if (isSequenceExpression(expression)) {
```

Comment:
What happens if `expression` is `undefined` or `null`, wouldn't you get `Cannot read property 'type' of undefined` when you call `isSequenceExpression`?

Reply:
Dang it, you're right. I've pushed another update.